packages/blade/src/tokens/theme/createTheme.ts

```diff
@@ -347,6 +347,7 @@ export const createTheme = ({ brandColor }: { brandColor: ColorInput }): ThemeTo
   const brandedThemeTokens = overrideTheme({
     baseThemeTokens: paymentTheme,
     overrides: {
+      name: `custom-${tinycolor(brandColor).toHex()}`,
```

Comment:
ideally we shall only allow hex values because hsl to hex doesn't give you the same color always there are discrepancies sometimes. but we can ignore for that but add this somewhere in code so we can come back later

Reply:
Our backend sends different formats 😢